styles/helpers/_layout.scss

```diff
@@ -1,3 +1,26 @@
+/* ==========================================================================
+   Margins & Padding
+   ========================================================================== */
+
+$props: ('margin', 'padding');
+$subProps: ('left', 'right', 'top', 'bottom');
+
+@each $p in $props {
```

Comment:
After talking with @AmeeMorris we've agreed to give helpers more space in the library. This adds margin and padding helpers. I need to add to this though;

- [ ] add '' to `subProps` so we get all round helpers
- [ ] what about breakpoints?

Reply:
This is cool!